extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_links.php

```diff
@@ -1,5 +1,5 @@
 <?php if ( !empty( $model['header'] ) ) : ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)